src/Tests/WarriorRogueMage/WarriorRogueMage.Tests/Behaviors/TestTalentsBehavior.cs

```diff
@@ -4,9 +4,7 @@
 //   subject to the Microsoft Permissive License. All other rights reserved.
 // </copyright>
 // <summary>
-//   Created by: Fastalanasa
-//   Date      : 1/30/2012 10:37:31 PM
-//   Purpose   : Testing the TalentsBehavior class.
+//   Testing the TalentsBehavior class.
```

Comment:
nit: Would be nice if these could follow the same pattern, like `Tests the TalentsBehavior class.`